test/java/seedu/addressbook/parser/NameIsSimilarTest.java

```diff
@@ -0,0 +1,15 @@
+package seedu.addressbook.parser;
+
+import seedu.addressbook.data.person.*;
+
+import org.junit.Before;
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+public class NameIsSimilarTest {
+
+    
+    
```

Comment:
Instead of adding the method you should try to call the method here first and let it fail.